src/WebJobs.Script.WebHost/Middleware/HostAvailabilityCheckMiddleware.Log.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Middleware
+{
+    public partial class HostAvailabilityCheckMiddleware
+    {
+        private static class Logger
+        {
+            private static readonly Action<ILogger, Exception> _hostUnavailableAfterCheck =
+               LoggerMessage.Define(LogLevel.Warning, new EventId(3, nameof(HostUnavailableAfterCheck)), "Host unavailable after check. Returning error.");
+
+            private static readonly Action<ILogger, Exception> _hostAvailabilityCheckSucceeded =
+               LoggerMessage.Define(LogLevel.Debug, new EventId(3, nameof(HostAvailabilityCheckSucceeded)), "Host availability check succeeded.");
```

Comment:
Are these EventIds right?

Reply:
Arrgh! Just talked about how I needed to fix this and forgot to do it before the merge :(

Reply:
Fixing